docs/development.md

```diff
@@ -24,6 +24,26 @@ $ npm run build
 
 After building is done, you can execute the application with `npm start`.
 
+### Packaging
+Package `dist/` directory as distributable formats with [`electron-builder`](https://github.com/electron-userland/electron-builder).
+Packages will be generated into `release/` directory.
+
+```
+$ npm run package:<all | windows | mac | linux>
+```
+
+#### Code signing
+To build trusted packages, need to set some environment variables.
```

Comment:
Maybe: "Set environment variables to build trusted packages" 

Just for consistency with other sections (Building, Packaging, ...) which start with an action word.